lib/hanami/cli/commands/db/rollback.rb

```diff
@@ -0,0 +1,39 @@
+module Hanami
+  class CLI
+    module Commands
+      module Db
+        # @since x.x.x
+        # @api private
+        class Rollback < Command
+          requires "model.sql"
+
+          desc "Rollback the database"
+
+          option :steps, desc: "Steps to rollback the database", default: 1
```

Comment:
In an effort to achieve a ubiquitous language I'd suggest to decide on a term for the concept of version/step and stick to that one term. That has the added benefit of being less confusing for the uninitiated 😄 .

Reply:
Also you could just use the generic `:num` as a name for this option then it reads better if you were to decide on `version` as the name for the concept.

Reply:
Thanks for the feedback ;) maybe I try to support 'hanami db rollback 2', then we can avoid use steps, version or whatever, but first I should implement default in arguments, what do you think.?